.travis.yml

```diff
@@ -37,11 +41,14 @@ before_script:
   - make install
   - popd
   - popd
-  - mkdir build
-  - cd build
+  - pip install --user -r requirements.txt
+
 
 script:
+  - mkdir build
+  - pushd build
   - cmake -DCMAKE_PREFIX_PATH=$INSTALL_ROOT -DUSE_RPATH=ON -DCMAKE_INSTALL_PREFIX=$INSTALL_ROOT ..
   - make install
-  - export PYTHONPATH=$PYTHONPATH:$INSTALL_ROOT/lib/python${PYTHON_VERSION}/site-packages
+  - export PYTHONPATH=$PYTHONPATH:$INSTALL_ROOT/lib/python${PYENV_VERSION}/site-packages
```

Comment:
Is setting the python path necessary?

Reply:
Well this test runs on the installed version of sunbeam - which I think is a good thing - and then the `PYTHONPATH` manipulations are required.

Reply:
Alright